src/js/which-transition-event.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * Find which transition event is supported.
+ *
+ * @returns {String|null} - Returns the supported transition event or `null`.
```

Comment:
would be better if the functionn returns `''` instead of `null` to keep type consistency